ci-operator/config/medik8s/node-maintenance-operator/medik8s-node-maintenance-operator-main__4.12.yaml

```diff
@@ -43,22 +39,52 @@ tests:
   skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
   steps:
     cluster_profile: medik8s-aws
-    dependencies:
-      OO_BUNDLE: my-bundle
     env:
       BASE_DOMAIN: medik8s-ci.projects.ecosystem.sysdeseng.com
-      OO_INSTALL_NAMESPACE: nmo-install
+      OPERATOR_NAMESPACE: nmo-install
     test:
+    - as: e2e-install-operator-last-version
+      cli: latest
+      commands: |
+        oc create namespace "$OPERATOR_NAMESPACE"
+
+        # workaround for OLM pod not running with restricted PSA
+        oc label --overwrite ns "$OPERATOR_NAMESPACE" security.openshift.io/scc.podSecurityLabelSync=false
+        oc label --overwrite ns "$OPERATOR_NAMESPACE" pod-security.kubernetes.io/enforce=privileged
+
+        # OPERATOR_NAMESPACE variable is used by bundle-run and bundle-run-update
+        BUNDLE_IMG=quay.io/medik8s/node-maintenance-operator-bundle:v0.17.0 make bundle-run
```

Comment:
nit
I think it might be easy to forget to update the last released version tag (here 0.17.0) and so only test the update from "some version in the past".

We might make this version stand up a bit more, either with some Env variable to be set, or just with a comment (or both :) )

Reply:
I agree, next time we add OCP 4.17 or release branch we should use this value as an env variable for better visibility.

Reply:
why not this time? :)